atomic_reactor/plugins/post_tag_and_push.py

```diff
@@ -76,24 +64,14 @@ def push_with_skopeo(self, registry_image, insecure, docker_push_secret,
         if insecure:
             cmd.append('--dest-tls-verify=false')
 
-        if not source_docker_archive:
-            # If the last image has type OCI_TAR, then hunt back and find the
-            # the untarred version, since skopeo only supports OCI's as an
-            # untarred directory
-            # OSBS2 TBD exported_image_sequence will not work for multiple platform
-            image = [x for x in self.workflow.data.exported_image_sequence if
-                     x['type'] != IMAGE_TYPE_OCI_TAR][-1]
-
-            if image['type'] == IMAGE_TYPE_OCI:
-                source_img = 'oci:{path}:{ref_name}'.format(**image)
-                cmd.append('--format=v2s2')
-            elif image['type'] == IMAGE_TYPE_DOCKER_ARCHIVE:
-                source_img = 'docker-archive://{path}'.format(**image)
-            else:
-                raise RuntimeError("Attempt to push unsupported image type %s with skopeo" %
-                                   image['type'])
+        if image['type'] == IMAGE_TYPE_OCI:
+            source_img = 'oci:{path}:{ref_name}'.format(**image)
```

Comment:
`get_exported_image_metadata` return value does not have key `ref_name`.

Reply:
it is added by `flatpak_create_oci`. Added a comment to clarify that.